Description: Reorder LDADD libraries
 The patch reorders the LDADD libraries to make the compiler aware of the
 needed libacl references.  GCC-9 uses the --as-needed flag by default which
 does not the detect the required references when using the old order.
Author: Arno Onken <asnelt@asnelt.org>
Bug-Debian: https://bugs.debian.org/925819
Index: rrep-1.3.6/src/Makefile.am
===================================================================
--- rrep-1.3.6.orig/src/Makefile.am
+++ rrep-1.3.6/src/Makefile.am
@@ -20,5 +20,5 @@ rrep_SOURCES = rrep.c messages.c bufferi
 noinst_HEADERS = bufferio.h messages.h pattern.h rrep.h
 
 AM_CPPFLAGS = -DLOCALEDIR=\"$(localedir)\" -I$(top_builddir)/lib -I$(top_srcdir)/lib
-LDADD = $(LIB_ACL) $(LIBICONV) $(LIBINTL) ../lib/libgnu.a
+LDADD = ../lib/libgnu.a $(LIB_ACL) $(LIBICONV) $(LIBINTL)
 
